packages/v4/src/content/design-guidelines/usage-and-behavior/modal/modal.md

```diff
@@ -32,25 +43,33 @@ Modal dialogs interrupt user task flow and should be avoided for any information
 
 ### Types of modals
 #### Get confirmation
-Use a confirmation dialog to validate user decisions and communicate consequences. Confirmation dialogs need to clearly restate the action the user is trying to take and explicitly outline the consequences of that action.
+Use a confirmation dialog to validate user decisions and communicate consequences. Confirmation dialogs need to clearly restate the action the user is trying to take and explicitly outline the consequences of that action. They should be clear, concise, and direct. Give the user the important information first—additional detail like “Are you sure?” clutters your message and can be distracting.
+
+Depending on your design template, follow this format for confirmation dialogs:
+* **Headline:** A headline is usually phrased as a question. Include important keywords (like “permanent”) in the headline.
+* **Body text:** Body text gives information about the action’s consequence.
+* **Buttons:** Buttons allow a user to answer the headline question. These can simply be “yes” or “no,” but adding some context increases clarity—especially when skimming.
 
 **Note:** Since confirmation dialogs disrupt user task flow, do not provide confirmation dialogs for actions that are easily reversed or insignificant.
 
 ##### Confirm a non-destructive action
-![Modal](./img/confirm-non-destructive-action.png)
+<img src="./img/confirm-non-destructive-action.png" alt="Confirm a non-destructive action" width="1100"/>
+
 Provide buttons on confirmation dialogs that make it simple for users to understand and confirm their choices. When choosing button labels, use descriptive verbs that clearly indicate the consequence of a selection, like Delete, Save, or Create. Avoid vague labels like OK or Done.
 
 Ambiguous title and button labels make it more difficult for the user to understand their options and make a decision. Descriptive title and button labels make it easy for the user to scan the content and make a decision quickly.
 
 ##### Confirm a destructive action
-![Modal](./img/confirm-destructive-action.png)
+<img src="./img/confirm-destructive-action.png" alt="Confirm a destructive action" width="1100"/>
```

Comment:
Hmmm... While I agree about the messaging, I actually like the button labeling of the first example better -- just Delete and Cancel and clear and to the point.

Reply:
Yeah, I think the buttons can remain the same on both (although I think both are fine)! I think a point that can be made here is that the "question" you are posing in the modal should be easily answered via the buttons. So maybe on the left example, you could change the title to "Are you sure you want to proceed", and have the buttons be "Delete" and "Cancel" to show the possible confusion. (Ie: a natural answer to "Are you sure you want to proceed" would be "Yes, proceed" or just "Proceed" and would not be "Delete") Does that make sense?

Reply:
The way I look at this, the affirmative action should always be labeled to reflect the result on clicking the button. In this case, Delete. While I agree with everything said about the messaging, the fact is that users often don't read. They will glance at this and either immediately click Delete or Cancel because they got here by mistake. Using labels like Yes or Proceed forces them to do more cognitive work by reading to understand what they are saying yes to.

Reply:
This is nitpicky, but could you expand the left modal so that the header fits on one line?

Reply:
The "Leave page without saving?" has a repetitive first line. The UX writing best practice is to have a heading (which we do), then the body text should get right to the point. Any extra questions distracts the user. So the photo should have: 

Leave page without saving? 
Any unused information will be lost.